player/Player.gd

```diff
@@ -43,3 +43,6 @@ func _physics_process(_delta) -> void:
 
 func _on_weapon_shoot(Bullet: PackedScene, _position: Vector2, _direction: Vector2) -> void:
 	emit_signal("shoot", Bullet, _position, _direction)
+
+func get_player_position() -> Vector2:
+	return global_position
```

Comment:
This function it's kinda useless (you can just do `player.global_position`) and I think you are not using it

Reply:
removed

Reply:
yea thats what i was doing already, I created the function in the beginning and forgot to remove